src/EndpointIdentifier.php

```diff
@@ -49,4 +54,9 @@ public function getUrl(): string
     {
         return "v{$this->version}/{$this->package}/{$this->apiAction}";
     }
+
+    private function checkVersionFormat(string $version): bool
```

Comment:
toto by som necheckoval, mozu si ludia chciet nechat tie verzie ako boli 1, 2, 3, atd. pripadne si tam dat akykolvek iny string ak chcu, nemusia to byt cisla

Reply:
pravda, ale ponechal som kontrolu aspon `/` kvoli routovaniu.